lib/interface.ts

```diff
@@ -3,16 +3,22 @@
 import restify = require('restify');
 import bunyan = require('bunyan');
 import BAPI = require('./blpapi-wrapper');
+import Session = require('./apisession/session');
 
 export interface IOurRequest extends restify.Request {
     clientCert?: any;
     blpSession: BAPI.Session;
+    apiSession?: Session;
+}
+
+export interface IBufferedData<T> {
```

Comment:
My concern is that the code that gives out objects adhering to this interface is not co-located with the interface itself.  It gives the impression that having the itnerface exported from `interface.ts` that `IBufferedData` can be used for multiple different APIs, when it is really just used for one.